majordodo-core/src/main/java/majordodo/replication/ReplicatedCommitLog.java

```diff
@@ -561,11 +560,19 @@ private void openNewLedger() throws LogNotAvailableException {
             writer = new CommitFileWriter();
             currentLedgerId = writer.getLedgerId();
             LOGGER.log(Level.INFO, "Opened new ledger:" + currentLedgerId);
+            // #160: workaround to prevent BookKeeper fault if a Bookie goes down when there are no entries on the ledger
+            writer.writeEntry(StatusEdit.NOOP());
```

Comment:
if this write fails it is better to try to delete the ledger from metadata (BookKeeper#deleteLedger)